drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenter.java

```diff
@@ -42,18 +42,21 @@
 
     private AtomicBoolean disabled = new AtomicBoolean(true);
 
-    private String factName = null;
+    /* In case of a selected column with <b>assigned</b> Instance, it contains its related Instance title.
+       Otherwise, in case of a <b>NOT assigned</b> Instance, it contains all the titles assigned to other columns of
+       the same group with the following syntax: "TITLE1;TITLE2;.." */
+    private String filterTerm = null;
```

Comment:
Here, I just renamed 'factName' to 'filterTerm'. Considering the description I put in the comment, it fits better, in my opinion.